gfx/video_crt_switch.c

```diff
@@ -200,14 +213,31 @@ static void switch_res_crt(
       videocrt_switch_t *p_switch,
       unsigned width, unsigned height, unsigned crt_mode, unsigned native_width, int monitor_index, int super_width)
 {
+   static char* current_core;
+   static char current_content_dir[PATH_MAX_LENGTH];
+
    unsigned char interlace = 0,   ret;
    const char* err_msg;
    int w = native_width, h = height;
    double rr = p_switch->ra_core_hz;
    
    if (crt_sr2_init(p_switch, monitor_index, crt_mode, super_width)) /* Checked SR2 is loded if not Load it */
    {
- 
+      /* Check for core and content changes in case we need to make any adjustments */
+      if(crt_switch_core_name())
+         current_core = crt_switch_core_name();
+      else
+         current_core = (char*)"\0";
+      fill_pathname_parent_dir_name(current_content_dir, path_get(RARCH_PATH_CONTENT), sizeof(current_content_dir));
+      if (strcmp(p_switch->core_name, current_core) || strcmp(content_dir, current_content_dir))
```

Comment:
Could you please use `string_is_equal()` instead of `strcmp()`?

(Just for consistency with the rest of the RA codebase)

Reply:
Sure, I made the change and tested to ensure it still works as before.

Changes are present on the just-amended commit.